Makefile

```diff
@@ -15,12 +16,21 @@ TESTING_CONFIG := $(CURDIR)/tests/data/testing_config.py
 build:
 	$(CONTAINER_ENGINE) build -f Containerfile -t $(IMAGE_NAME) .
 
-re-build:
+rebuild:
 	$(CONTAINER_ENGINE) build --no-cache --pull=true -f Containerfile -t $(IMAGE_NAME) .
 
 run:
-	$(CONTAINER_ENGINE) run -it --rm -v $(CURDIR):/config:z -v $(SECRETS_DIR):/secrets:z --env TOKMAN_CONFIG=/config/config.py --env WORKERS=$(WORKERS) --env LOG_LEVEL=$(LOG_LEVEL) --publish 8000 tokman
+	$(CONTAINER_ENGINE) run -it --rm -v $(CURDIR):/config:z -v $(SECRETS_DIR):/secrets:z --env TOKMAN_CONFIG=/config/config.py --env WORKERS=$(WORKERS) --env LOG_LEVEL=$(LOG_LEVEL) --publish 8000 $(IMAGE_NAME)
+
+build-test-image:
+	$(CONTAINER_ENGINE) build -f Containerfile.tests -t $(TEST_IMAGE) .
+
+rebuild-test-image:
+	$(CONTAINER_ENGINE) build --no-cache --pull=true -f Containerfile.tests -t $(TEST_IMAGE) .
```

Comment:
This is not necessarily bad, but the other projects don't have such a target, so you are making things diverge again :)

Reply:
My thought was that we have 'build' and 'rebuild' targets, so we should
also have build-test-image and rebuild-test-image for symmetry.

Reply:
I see. We can keep it then.

Just be aware that `--pull=true` is not needed in this case, as the test image is built on top of a local image, so there is nothing to pull.